system/elastiflow/values.yaml

```diff
@@ -351,9 +351,3 @@ kibana:
   #     memory: 4Gi
   #   limits:
   #     memory: 5Gi
-
-# Deploy Prometheus alerts.
-alerts:
-  enabled: false
-  # Name of the Prometheus to which the alerts should be assigned to.
-  prometheus: infra-frontend
```

Comment:
This part was flagged by the CI:
```
system/elastiflow/values.yaml
  354:1     error    duplication of key "alerts" in mapping  (key-duplicates)
```
I merged it with the functionally identical section at the top of the file.